staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -272,8 +274,15 @@ func (s *Controller) processNextWorkItem() bool {
 		return true
 	}
 
-	runtime.HandleError(fmt.Errorf("error processing service %v (will retry): %v", key, err))
-	s.queue.AddRateLimited(key)
+	var re *api.RetryError
```

Comment:
I might have missed something, I don't see `RetryError` added to `k8s.io/cloud-provider/api`, maybe you forgot to push the commit for this change?

Reply:
Ah yes indeed 🤦 thanks for pointing out, I just pushed the missing file.